Gemfile

```
@@ -37,6 +37,7 @@ group :development do
   gem 'spring' # Spring speeds up development by keeping your application running in the background.
   gem 'spring-commands-rspec' # This gem implements the rspec command for Spring.
   gem 'spring-watcher-listen', '2.0.1' # Makes Spring watch the filesystem for changes using Listen
+  gem "debug"
```

```suggestion
  gem 'debug'
```

There are a few discussions if we should not just use `"` everytime instead of `'` when string without string interpolation 🤔 
But as for now, the convention remains using single quote whenever possible 😇

---

Also, if you run the project with `rails s -p 3000 -b '0.0.0.0'` (when not in need to watch css or migrate the db), you can use `binding.pry` anywhere in the code (even in tests) and then have a quite nice CLI interface for step by step debug & calling any method/variable in the same way you would do in the `rails console` ;-)

---

I removed this gem in 3301251b because we already had `byebug`